content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+In machine learning projects, number of <abbr>experiments</abbr> grow rapidly.
+DVC can track these experiments, list their most relevant parameters and
+metrics, and commit only the ones that we need to Git.
+
+In this section, we explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+<details>
+
+## ⚙️ Installing the Example Project
+
+These commands are run in the [`get-started-experiments`][gse] project. You can
+run the commands in this document after cloning the repository and installing
+the requirements.
+
+### ⚡ Clone the project and create virtual environment
+
+Please clone the project and create a virtual environment.
+
+> We create a virtual environment to keep the libraries we use isolated from the
+> rest of your system. This prevents version conflicts.
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments -b get-started
+$ cd get-started-experiments
+$ virtualenv .venv
```

Comment:
it won't work on Mac (need to check Windows) - you need to provide Python 3

I would simplify this with `We strongly recommend creating a virtual environment first.` - the way we handle this in the current example get started

Reply:
`virtualenv` requires a separate installation in all systems. The standard one in Py3 is `venv`. We discussed this in https://github.com/iterative/example-repos-dev/issues/26 and decided to continue with `virtualenv`. 

I can also strongly recommend, if that's the point :)

Reply:
Even `venv` requires a separate `apt install python3-venv` in Debian and derivatives.

Reply:
My question was about not providing so much details. We can keep `git clone` and maybe `pip install -r requirements` ... we can put a link nearby to the virtualenv like we do in the current get started?